### DISCUSSION TITLE
Guide to running common neural networks on Yocto

### DISCUSSION THREAD

**Discussion by mozcelikors** [OE4T/meta-tegra#379] [Ideas]
Hi,
Is there a guide to running common pretrained neural networks (YOLOv3, RCNN etc) on Yocto, Jetson boards? I know a few guides exist for Ubuntu on Jetson boards, but I would like to evaluate them on Yocto-based OS. If such guides or recipes exist, it would be good -imho- to compile them in the Wiki section.
Thanks,

**Comment by madisongh** [comment]
I don't know of one... contributions are welcome!

As a short cut to getting started, you should be able to run the Jetson-compatible containers in the [NVIDIA NGC catalog](https://ngc.nvidia.com/catalog/all) on a Yocto-based build that includes nvidia-docker support. But recipes for building the various frameworks don't seem to be readily available.

**Comment by dwalkes** [comment]
We've shared a repo at https://github.com/BoulderAI/deepstream-nvdsanalytics which you can use to run with NVIDIA's deepstream samples by building/running with docker on yocto based builds in case that's useful.
